### DISCUSSION TITLE
workflow_types.go Arguments in Template is shown DEPRECATED, what keyword should I use to replace it for argument assignment？

### DISCUSSION THREAD

**Discussion by liujiong1982** [argoproj/argo-workflows#7552] [Q&A] [answered]
workflow_types.go 
Arguments in Template is shown DEPRECATED, what keyword should I use to replace it for argument assignment？

**Comment by terrytangyuan** [comment, marked as answer]
I am not seeing a deprecation note on arguments in `workflow_types.go`. Could you point me to where you are seeing this?

**Reply by liujiong1982** [reply]
> i
![image](https://user-images.githubusercontent.com/7462724/149414022-bda8a184-b4c2-485d-b6ab-78ab181ca186.png)
Here you go

**Reply by terrytangyuan** [reply]
You are using a very old version.  Use inputs for newer versions of Argo Workflows: https://github.com/argoproj/argo-workflows/blob/master/pkg/apis/workflow/v1alpha1/workflow_types.go#L516
